tests/contrib/django/test_django_snapshots.py

```diff
@@ -284,3 +289,90 @@ def test_templates_disabled():
         resp = client.get("/template-view/")
         assert resp.status_code == 200
         assert resp.content == b"some content\n"
+
+
+@pytest.mark.skipif(django.VERSION < (3, 2, 0), reason="Only want to test with latest Django")
```

Comment:
How long will it be true that this mark filters to the "latest" Django version?

Reply:
Dunno. But it's in line with other tests like it. I guess they could be renamed to "latest Djangos" in other PR.